projects/samples/devices/controllers/position_sensor/position_sensor.c

```diff
@@ -76,8 +77,8 @@ int main(int argc, char **argv) {
 
     wb_motor_set_velocity(left_motor, -speed);
     wb_motor_set_velocity(right_motor, -speed);
-    ANSI_CLEAR_CONSOLE();
```

Comment:
I would keep this to ease the reading of the values.

Reply:
oh yes that was for my debugging ! sorry !